libs/core/src/lib/micro-process-flow/components/micro-process-flow/micro-process-flow.component.ts

```diff
@@ -0,0 +1,246 @@
+import {
+    AfterContentInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChildren,
+    ElementRef,
+    Input,
+    OnInit,
+    OnDestroy,
+    Optional,
+    QueryList,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { ContentDensityService, RtlService } from '@fundamental-ngx/core/utils';
+import { Subscription } from 'rxjs';
+import { startWith } from 'rxjs/operators';
+import { MicroProcessFlowItemComponent } from '../micro-process-flow-item/micro-process-flow-item.component';
+
+@Component({
+    selector: 'fd-micro-process-flow',
+    templateUrl: './micro-process-flow.component.html',
+    styleUrls: ['./micro-process-flow.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        class: 'fd-micro-process-flow',
+        '[class.fd-micro-process-flow--independent-steps]': 'independentSteps === true',
+        '[class.fd-micro-process-flow--compact]': 'compact === true'
```

Comment:
it should work like this
```suggestion
        '[class.fd-micro-process-flow--independent-steps]': 'independentSteps',
        '[class.fd-micro-process-flow--compact]': 'compact'
```